main.ts

```diff
@@ -90,13 +107,39 @@ async function ensureNscloudToken() {
   await exec.exec("nsc auth exchange-github-token --ensure=5m");
 }
 
-async function remoteNscBuilderExists(): Promise<boolean> {
+async function nscBuilderStatus(): Promise<boolean> {
+  const { stdout, stderr } = await exec.getExecOutput(
+    `nsc docker buildx status --output=json`,
+    null,
+    { ignoreReturnCode: true, silent: true }
+  );
+  
+  const parsed = JSON.parse(stdout)
+  if (!parsed || !Array.isArray(parsed)) {
+    return false
+  }
+  
+  const elems = <Array<any>>(parsed)
+  if (elems.length == 0) {
+    return false
+  }
+  
+  for (const elem of elems) {
```

Comment:
I wonder if it'd be more robust to reverse this check and return `true` if and only if `Status = success`

Reply:
done - we are accepting status Starting, Running, ServerSideProxy now because all of these mean that nsc-remote has been configured successfully or is being configured.